src/main/java/monique/command/DeleteCommand.java

```diff
@@ -10,23 +10,23 @@ public class DeleteCommand extends Command {
 
     private final int taskNum;
 
-    public DeleteCommand(int taskNum){
+    public DeleteCommand(int taskNum) {
         super();
-        this.taskNum= taskNum;
+        this.taskNum = taskNum;
     }
 
     @Override
     public void execute(TaskList tasks, Ui ui, Storage storage) throws DeleteException {
-        if (this.taskNum > tasks.getNumItems() -1 || this.taskNum < 0){
+        if (this.taskNum > tasks.getNumItems() - 1 || this.taskNum < 0) {
             throw new DeleteException();
         }
         Task deletedTask = tasks.getTask(this.taskNum);
         tasks.deleteTask(this.taskNum);
```

Comment:
Nice job on ensuring the task deletion logic is handled properly. The boundary checks before deleting a task are clear and avoid potential issues with invalid task numbers.